index.js

```diff
@@ -0,0 +1,17 @@
+function receivesAFunction(callback) {
+    return callback();
+}
+
+function callback(){
+}
```

Comment:
I don't think this is needed. 
```suggestion
function callback(){
}
```